source/geometries/Next100SiPM.cc

```diff
@@ -52,13 +54,16 @@ G4ThreeVector Next100SiPM::GetDimensions() const
 void Next100SiPM::Construct()
 {
   // ENCASING //////////////////////////////////////////////
-
+  //dimensions are increased in case of SiPM coating
   G4String sipm_name = "SIPM_S13372";
 
   G4double sipm_width  = 3.0 * mm;
   G4double sipm_length = 4.0 * mm;
   G4double sipm_thickn = 1.3 * mm;
 
+  if(with_coating_) sipm_thickn = sipm_thickn + coating_thickn_;
```

Comment:
I'm not sure I understand the purpose of this. If you do `if (!with_coating_) coating_thickn_ = 0;`, then you can use 

`sipm_thickn = sipm_thickn + coating_thickn_;`

for both cases.

Reply:
I agree, I'll adapt it.